lib/parallel_tests/tasks.rb

```diff
@@ -135,6 +135,8 @@ def configured_databases
       end
 
       def for_each_database(&block)
+        return unless defined?(ActiveRecord)
+
         # Use nil to represent all databases
         block&.call(nil)
```

Comment:
needs to be after this to preserve original behavior ?

Reply:
Oh gosh, nice catch 😄 https://github.com/grosser/parallel_tests/pull/930/commits/0f8e6a830c9e7fbd05fd70d5f9145173838f6f07